spec/database_helper.rb

```diff
@@ -0,0 +1,36 @@
+require "sequel"
+
+class DatabaseHelper
+  USER = ENV.fetch("DATABASE_USER")
```

Comment:
Maybe we could default to root and no pass (possibly a default test-db as well), like we do in the documentdb tests. If we can live without `.env` for testing I think that would be great. Not sure what the common Twingly opinion is though. :)